### ISSUE TITLE
Thoughts about WCAG and primarily color contrasts

### ISSUE THREAD

**Issue by hampusn** [sitevision/envision#87]
Many of the components does not fulfill requirements in WCAG. A quick scan I've made makes me think that it's mainly about color contrasts.

What are your thoughts about WCAG and this library?

Since it's a standalone library I'll acknowledge that it shouldn't necessarily be affected by other products (such as SiteVision) and that WCAG therefore falls out of scope. But since Envision is bundled with SiteVision, I believe from SiteVision's point of view, SiteVision would benefit from Envision being 'WCAG-safe'. :)

Components found with insufficient color contrasts:
* http://envisionui.io/components/badge/
* http://envisionui.io/components/button/
* http://envisionui.io/components/button-group/
* http://envisionui.io/components/dropdown/
* http://envisionui.io/components/embedded/
* http://envisionui.io/components/feed/
* http://envisionui.io/components/form/
* http://envisionui.io/components/modal-dialog/
* http://envisionui.io/components/navigation/
* http://envisionui.io/components/news-item/
* http://envisionui.io/components/pagination/
* http://envisionui.io/components/popover/
* http://envisionui.io/components/progress/
* http://envisionui.io/components/tab/
* http://envisionui.io/components/table/

_Most_ of these fails due to the color contrast in the Button component, which many of the other components use. But there are still some other issues with different shades of grey on text as seen in the Embedded component.

Most contrasts can probably be solved by tinkering with the colors in variable.scss:

https://github.com/sitevision/envision/blob/5944c8451c13affc3ee784314b64496c80d37602/src/scss/variable.scss#L76-L107

This issue is related to #71
